### PR TITLE
[20.09] python3Packages.nixpkgs: unmark as broken

### PR DESCRIPTION
**Pull request by t184256**
###### Motivation for this change

I have no idea why it got marked broken (the same day I've checked it's OK), it's working fine.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by FRidh** [comment]
Check all Python versions.

**Comment by t184256** [comment]
@FRidh: oh, I see now!

pythonix dependency looks broken (builds, doesn't import) on all versions but 3.8, what should I do wrt that, mark pythonix conditionally broken and unmark this one? disable pythonix conditionally? disable this one conditionally?

cc @Mic92

File: pkgs/development/python-modules/nixpkgs/default.nix [hunk 1]
```diff
@@ -24,7 +24,6 @@ buildPythonPackage rec {
     homepage = "https://github.com/t184256/nixpkgs-python-importer";
     license = licenses.mit;
     maintainers = with maintainers; [ t184256 ];
-    broken = true;
   };
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/nixpkgs/default.nix:28]:
```suggestion
```

Comment (t184256) [reply pkgs/development/python-modules/nixpkgs/default.nix:28]: sorry, huh?

**Comment by Mic92** [comment]
@GrahamcOfBorg build python3Packages.nixpkgs

**Comment by t184256** [comment]
Folks, I still am not sure what to do with that one. Guidance would be welcome.

**Comment by mweinelt** [comment]
Unfortunately NixOS 20.09 has reached its end-of-life status on 2021-07-01, one
month after the release of NixOS 21.05.

Since we do not accept any changes to its branches anymore, I'm closing this pull
request.

**Pull request closed without merging**